Sources/Models/Sender.swift

```diff
@@ -25,33 +25,33 @@
 import Foundation
 
 /// An object that groups the metadata of a messages sender.
-@available(*, deprecated: 3.0.0, message: "`Sender` has been replaced with the `SenderType` protocol in 3.0.0")
-public struct Sender: SenderType {
+public struct Sender {
 
     /// MARK: - Properties
 
     /// The unique String identifier for the sender.
     ///
     /// Note: This value must be unique across all senders.
-    public let senderId: String
-
-    @available(*, deprecated: 3.0.0, message: "`id` has been renamed `senderId` as defined in the `SenderType` protocol")
-    public var id: String {
-        return senderId
-    }
+    public let id: String
 
     /// The display name of a sender.
     public let displayName: String
 
     // MARK: - Intializers
 
-    public init(senderId: String, displayName: String) {
-        self.senderId = senderId
```

Comment:
Please revert all these changes to `Sender`, outside scope of PR and breaks Sender from being SenderType